### PR TITLE
style: reword universe playbook prompts

### PR DESCRIPTION
**Pull request by ghost**
## Description

Reworded the NVIDIA library EULA warning prompt and the name of NVIDIA library installation skip warning task for improved clarity.

Signed-off-by: Lalith Vipulananthan <lalith.vipulananthan@tier4.jp>

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: ansible/playbooks/universe.yaml [hunk 1]
```diff
@@ -9,15 +9,15 @@
   vars_prompt:
     - name: install_nvidia
       prompt: |-
-        [Warning] Some components depend on NVIDIA libraries (CUDA, cuDNN, TensorRT), which require license agreements.
-        Did you confirm the licenses and are you sure to to install them? [y/N]
+        [Warning] Some Autoware components depend on the CUDA, cuDNN and TensorRT NVIDIA libraries which have end-user license agreements that should be reviewed before installation.
+        Proceed? [y/n]
```
Comment (kenji-miyake) [inline ansible/playbooks/universe.yaml]: Same as https://github.com/autowarefoundation/autoware/pull/201#discussion_r854146571.

Comment (ghost) [reply ansible/playbooks/universe.yaml]: Okay. Again, is it okay to revert the change with another commit?

Comment (kenji-miyake) [reply ansible/playbooks/universe.yaml]: Yes, of course.

Comment (kenji-miyake) [reply ansible/playbooks/universe.yaml]: @LalithVipulananthan I reverted it. :bow: 

But is `Proceed?` a bit ambiguous? :thinking: 
Regardless of the answer, the installation itself will proceed.

Comment (ghost) [reply ansible/playbooks/universe.yaml]: I had to resolve a merge conflict, and in the process I updated the wording to say `Install NVIDIA libraries? [y/N]`. Does that work better?

Comment (kenji-miyake) [reply ansible/playbooks/universe.yaml]: Yeah, perfect!

**Review by kenji-miyake — APPROVED** [review]

**Pull request merged**